docs/Tutorials/DomainCreation.md

```diff
@@ -184,28 +184,28 @@ are:
 ```
 
 //Folding highest cube downward:
-{{53,54,69,70},{53,54,40,41}},
+{{53,54,69,70},{53,54,40,50}},
 {{53,57,69,73},{53,57,52,56}},
 {{57,58,61,62},{57,58,73,74}},
 {{54,58,70,74},{54,58,55,59}},
 
 //Folding cross cubes together:
-{{54,55,38,39},{54,41,38,25}},
+{{54,55,38,39},{54,41,38,34}},
 {{58,59,42,43},{58,62,42,46}},
-{{53,52,37,36},{53,40,37,33}},
+{{53,52,37,36},{53,49,37,33}},
 {{57,56,41,40},{57,61,41,45}},
 
 //Folding second lowest cube upward:
 {{38,42,39,43},{38,42,22,26}},
-{{33,34,37,38},{21,22,25,26}},
+{{33,34,37,38},{21,22,37,38}},
 {{36,37,40,41},{21,37,25,41}},
 {{41,42,45,46},{41,42,25,26}},
 
 //Folding bottom cube around domain:
 {{33,34,49,50},{21,22,5,6}},
 {{39,43,55,59},{22,26,6,10}},
 {{45,46,61,62},{25,26,9,10}},
-{{36,40,52,56},{21,25,5,8}},
+{{36,40,52,56},{21,25,5,9}},
 {{5,6,9,10},{69,70,73,74}}
 
 ```
```

Comment:
Thanks for checking! One quick way of verifying that the corners are incorrect is to check `a-b==c-d`, where those correspond to the corner numbers in `{{a,b,c,d}}`.

i.e., for `{{54,55,38,39},{54,41,38,34}}`, you can check that 38-39==54-55 and 54-41!=38-34, so the second face fails the check and should be examined. This is because the corners are ordered and edges are parallel, so the difference in corner values between edges is the constant. If you'd like to add a comment to this effect on the documentation I'd appreciate it :+1: otherwise I can add it myself after this PR gets in.

Reply:
Will do! I should really have copied the whole text instead of trying to note down which corners need fixing. The latter obviously didn't work out for me. ^^